src/engine/websites/Desu.ts

```diff
@@ -0,0 +1,33 @@
+import { Tags } from '../Tags';
+import icon from './Desu.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+
+const scriptPages = `
+    new Promise((resolve, reject) => {
+        setTimeout(() => {

```

Comment:
I think `PagesSinglePageJS` accepts an optional parameter to delay the script injection, therefore `setTimeout` might not be necessary